Assets/PatchKit Patcher/Editor/CustomBuilding.cs

```diff
@@ -78,6 +79,29 @@ private static string PatcherExecutableName(BuildTarget target)
             }
         }
 
+        public static PlatformType BuildTargetToPlatformType(BuildTarget target)
+        {
+            switch (target)
+            {
+                case BuildTarget.StandaloneWindows:
+                case BuildTarget.StandaloneWindows64:
+                    return PlatformType.Windows;
+
+                case BuildTarget.StandaloneLinux:
+                case BuildTarget.StandaloneLinux64:
+                case BuildTarget.StandaloneLinuxUniversal:
+                    return PlatformType.Linux;
+
+                case BuildTarget.StandaloneOSXIntel:
+                case BuildTarget.StandaloneOSXIntel64:
+                case BuildTarget.StandaloneOSXUniversal:
+                    return PlatformType.OSX;
+
+                default:
+                    throw new System.ArgumentException("target");
```

Comment:
At `default` it is always useful to pass the value. Then by looking at it you will immediately know what is missing.